src/main/resources/application.yml

```diff
@@ -145,7 +145,7 @@ doc_assembly:
   oconFormOrganisationWithoutDQs: "CV-CMC-GOR-ENG-0015.docx"
   noticeOfTransferToCcbcSentToDefendantTemplateId: "CV-CMC-GOR-ENG-0016.docx"
   paperResponseAdmissionTemplateId: "CV-CMC-GOR-ENG-0016.docx"
-  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630.docx"
+  defendantPinLetterTemplateID: "CV-CMC-LET-ENG-00630-1.odt" # TODO change to docx format
```

Comment:
this TODO seems to be pending.

Reply:
This was added by me. There was a water mark when creating a docx document from my machine.